test/userapp/index.js

```diff
@@ -335,20 +352,20 @@ const MixedInputFields = {
   noun: 'mixedinputfields',
   list: {
     display: {
-      label: 'mixed input fields',
+      label: 'Mixed Input Fields',
       description:
         'input fields are static, a sync function, and an async promise'
     },
     operation: {
       inputFields: [
         { key: 'key 1' },
         (z, bundle) => Promise.resolve({ key: bundle.key2 }),
-        Promise.resolve({ key: 'key 3' })
```

Comment:
This was allowed before and isn't anymore?

Reply:
I'm not sure if we ever allowed this before. But I'm pretty sure we don't allow this now as it won't even pass the schema validation.